src/legacy/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -128,19 +140,42 @@ class VisEditorVisualization extends Component {
       this.setState((prevState) => {
         const newHeight = prevState.height + (keyCode === keyCodes.UP ? -15 : 15);
         return {
-          height: Math.max(MIN_CHART_HEIGHT, newHeight)
+          height: Math.max(MIN_CHART_HEIGHT, newHeight),
         };
       });
     }
   }
 
+  hasShowPanelIntervalValue() {
+    const type = get(this.props, 'model.type', '');
+
+    return [
+      'metric',
+      'top_n',
+      'gauge',
+      'markdown'
+    ].includes(type);
+  }
+
+  getFormattedPanelInterval() {
+    const interval = get(this.props, 'model.interval', 0);
+
+    if (this.hasShowPanelIntervalValue() && interval) {
```

Comment:
if interval is an empty string we are skipping this but I still see anyway data rendered on the chart without knowing what interval I'm currently used.
Is the default the `auto`? if so, please change it to the previous `get` method.